README.md

```diff
@@ -1,32 +1,30 @@
-<a href="https://github.com/camaraproject/EdgeCloud/graphs/contributors" title="Contributors"><img src="https://img.shields.io/github/contributors/camaraproject/EdgeCloud?style=plastic"></a>
-<a href="https://github.com/camaraproject/EdgeCloud" title="Repo Size"><img src="https://img.shields.io/github/repo-size/camaraproject/EdgeCloud?style=plastic"></a>
-<a href="https://github.com/camaraproject/EdgeCloud/blob/main/LICENSE" title="License"><img src="https://img.shields.io/badge/License-Apache%202.0-green.svg?style=plastic"></a>
+# Edge Cloud
 
-# EdgeCloud
-Repository to describe, develop, document and test the EdgeCloud API family
+Repository to describe, develop, document, and test the §repo_name§ API family
 
 ## Scope
-* Service APIs for “Edge Cloud” (see APIBacklog.md)  
+
+* Service APIs for “Edge Cloud” (see APIBacklog.md)
 * It provides the customer with the ability to:  
-  * Provide and manage application images to be deployed on resources within the operator network 
-  * Use reserved compute resources within the operator network for the deployment of applications on VMs or containers  
-  * Influence the traffic routing from the user device toward the Edge instance of the Application 
+  * Discover the closest edge cloud zone to a given device.
+  * Provide and manage application images to be deployed on resources within the operator network. 
+  * Use reserved compute resources within the operator network for the deployment of applications on VMs or containers.  
+  * Influence the traffic routing from the user device toward the Edge instance of the Application. 
   * NOTE: The scope of this API family should be limited (at least at a first stage) to 4G and 5G.  
 * Describe, develop, document and test the APIs (with 1-2 Telcos)  
 * Started: 5th July 2022
-* Location: virtually  
-
-## Meetings
-* Meetings are held virtually
-* Schedule: Tuesdays 16h-17 CET, every two weeks. 
-* Meeting link: [LFX Zoom](https://www.google.com/url?q=https://zoom-lfx.platform.linuxfoundation.org/meeting/94237809115?password%3D05fb6d8a-a913-47d8-b003-db75ecdaa5d9&sa=D&source=calendar&ust=1711304713775725&usg=AOvVaw2KsTGn2S2i4Bu5V-nusuUI) 
-## Results
-* Sub Project is in progress
-
-## Contributorship and mailing list
-* To subscribe / unsubscribe to the mailing list of this Sub Project and thus be / resign as Contributor please visit <https://lists.camaraproject.org/g/sp-edc>.
-* A message to all Contributors of this Sub Project can be sent using <sp-edc@lists.camaraproject.org>.
-* CAMARA sp-edge-cloud: [Slack Channel](https://camara-project.slack.com/archives/C062PJNGW9F)
+
+# Contributing
+
+* Meetings
+  * Meeting link: [Registration](https://www.google.com/url?q=https://zoom-lfx.platform.linuxfoundation.org/meeting/94237809115?password%3D05fb6d8a-a913-47d8-b003-db75ecdaa5d9&sa=D&source=calendar&ust=1711304713775725&usg=AOvVaw2KsTGn2S2i4Bu5V-nusuUI)
```

Comment:
```suggestion
  * Meeting link: [Registration](https://zoom-lfx.platform.linuxfoundation.org/meeting/94237809115?password=05fb6d8a-a913-47d8-b003-db75ecdaa5d9)
```

Reply:
Good catch, thanks - fixed in latest commit